developer-resources

```diff
@@ -1 +0,0 @@
-developer-resources
```

Comment:
why did you delete the directories/symlinks? developer-resources and deploy?

Reply:
Those were sym directories that I think I accidentally added in my last release. The developer-resources contained a copy of some really old changes I made when I first started at Neo. So I removed them.